mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestDerivateContents.java

```diff
@@ -313,7 +313,7 @@ public Response getFileOrDirectoryMetadata() {
             .lastModified(Date.from(fileAttributes.lastModifiedTime().toInstant()))
             .header(HttpHeaders.CONTENT_LENGTH, fileAttributes.size())
             .tag(getETag(fileAttributes))
-            .header("Digest", getDigestHeader(fileAttributes.digest().toHexString()))
+            .header("Content-Digest", getDigestHeader(fileAttributes.digest().toHexString()))
```

Comment:
we could use fileAttributes.digest() here directly as a parameter to get the header value. This will support directly SHA-512 in the future, when we move to OCFL.

the format of the value will change to:
Before: MD5={checksum}
After: md5=:{checksum}:
So small letters for the algorithm and the actual value is wrapped in colons (:)
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Repr-Digest

Reply:
Sorry, confused myself her for a second about which header to use and the formatting. Should be correct now.

Not sure what you mean by "use `fileAttributes.digest()` here directly", since that method returns a `MCRDigest` object which is certainly not directly usable as a header value.

Reply:
I think you mean that we should account for the fact, that `MCRFileAttributes#digest()`will not necessarily return a MD5 digest, correct?